src/ui/profile_preferences.ui

```diff
@@ -228,9 +220,9 @@
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_miscellaneous">
          <property name="title">
-          <string>Misc</string>
+          <string>Miscellaneous</string>
```

Comment:
Why was this changed as part of this PR?

Reply:
It caught my eye in passing whilst I was working next to it on the preferences form, in English "Misc" is not *actually* a word - it is an abbreviation but it does not have the needed `.` for that and there is more than enough space for the full word.

Reply:
It seems fine to me, doesn't hurt anyone and keeps it a bit casual